Makefile.am

```diff
@@ -773,6 +773,7 @@ libvmdisplay_la_SOURCES = clients/vmdisplay/vmdisplay-parser.c \
 		clients/vmdisplay/vmdisplay.c		\
 		clients/vmdisplay/wayland-drm-protocol.c
 
+bin_PROGRAMS += vmdisplay-wayland
```

Comment:
Starting with 6.0, we don't use autotools anymore. We only use meson. But I will take this change for one last time. Please build via meson in the future